dashboard/modules/job/job_manager.py

```diff
@@ -269,7 +284,30 @@ def _get_current_node_resource_key(self) -> str:
                         return key
         else:
             raise ValueError(
-                "Cannot found the node dictionary for current node.")
+                "Cannot find the node dictionary for current node.")
+
+    def _handle_supervisor_started(self, job_id: str,
```

Comment:
this is doing both good and exception case for supervisor starting up, maybe change it to `_handle_supervisor_startup` instead of started that implies we've already started the actor ?

Reply:
true

Reply:
let me add header too